src/screens/NewProfile/NewProfile.js

```diff
@@ -156,12 +157,13 @@ class NewProfile extends React.Component<Props, State> {
 
   constructor(props: Props) {
     super(props);
-    const { apiUser } = props;
+    const { apiUser, importedWallet } = props;
     const value = apiUser && apiUser.username ? { username: apiUser.username } : null;
     const inputDisabled = !!(apiUser && apiUser.id);
+    const showRightPlaceholder = !importedWallet;
```

Comment:
If I imported my old mnemonic that has no user associated on the Pillar platform, then it will show my the 'Pick a username' field without placeholder.

Reply:
There was a discussion about this whether to show it to new pillarwallet users only who will get a smart account and ens upon registration. We can discuss this further on Monday

Reply:
Just confirmed that imported wallets from external source have no smartwallet feature yet (they need to init & deploy)

Reply:
Right! Then everything is correct :)